src/models/layout/mod.rs

```diff
@@ -109,17 +109,13 @@ mod test {
     fn parse_datetime_test() {
         let time_str = "1970-01-01T00:00:00Z".to_string();
         let parsed_dt = parse_datetime(&time_str[..]).unwrap();
-        let dt = Utc.from_utc_datetime(
-            &NaiveDateTime::from_timestamp_opt(0, 0).unwrap(),
-        );
+        let dt = DateTime::from_timestamp(0, 0).unwrap();
         assert_eq!(parsed_dt, dt);
     }
 
     #[test]
     fn format_datetime_test() {
-        let dt = Utc.from_utc_datetime(
-            &NaiveDateTime::from_timestamp_opt(0, 0).unwrap(),
-        );
+        let dt = DateTime::from_timestamp(0, 0).unwrap();
```

Comment:
Can you explain why you're switching this?  (I am not a Rust expert.)

Reply:
The `NaiveDateTime::from_timestamp_opt` API was deprecated. This upset the linter (clippy) and is now blocking some Dependabot PRs.

`DateTime::from_timestamp` is something relatively new (September 2023, it seems) and was recommended by the deprecation warning. Both essentially achieve the same function of getting the Unix epoch in a `DateTime<Utc>` structure. 

The original used a "naive" date time structure (basically date time with no time zone information) and passed it to a function that added the time zone information. The new API does all of this at once since the default timezone is UTC.

P.S. I found that there is a constant that points to `UNIX_EPOCH`, so I will update changes to use this to make it clearer.